libs/gretty-runner-tomcat/build.gradle

```diff
@@ -1,14 +1,8 @@
-import org.gradle.util.VersionNumber
-
 apply from: rootProject.file('common.gradle')
 
 dependencies {
   compile project(':libs:gretty-runner')
   compile "org.slf4j:log4j-over-slf4j:$slf4j_version"
-  // These dependencies are needed for compiling gretty-runner-tomcat.
-  // They are replaced by "real" dependencies at runtime.
```

Comment:
Why are these comments removed? Are they not valid any more?

Reply:
I think that the comments did not provide any value. What is a "real" dependency anyway?

Yes, the whole concept of "pick one reference version of Tomcat and hope that the code 
 works across several major releases such that we do not need to duplicate it" still exists, but the comment is nowhere near explaining that.

To me the changed implementation scope of "compileOnly" summarises the prior comment pretty well.